subsys/bootloader/debug/Kconfig

```diff
@@ -5,12 +5,14 @@
 #
 
 config SECURE_BOOT_DEBUG
+	default n if SOC_SERIES_NRF51X
+	default y
```

Comment:
Does SECURE_BOOT_DEBUG work with nrf51? If not we should use depends, not default.

Reply:
works with SEGGER but not UART, so there could optionally be a depend on !SOC_SERIES_NRF51X on the uart option.